scripts/bundled_installer

```diff
@@ -10,6 +10,9 @@
 export PYTHON_VERSION="3.7.2"
 export PYENV_ROOT="$HOME/.pyenv"
 export PYENV_BIN="$PYENV_ROOT/versions/$PYTHON_VERSION/bin"
+export SURPRESS_PATH_EXPORT_MESSAGE="true"
```

Comment:
You don't need to export this statement. Since you are `source`-ing `python_installer`, you can pass this as an argument:

```sh
         source ./python_installer SURPRESS_PATH_EXPORT_MESSAGE=true
```

Also,:
- the value can be a boolean rather than a string
- move this assignment into the `install_python` function defined below to keep its scope limited.